torch/onnx/symbolic_helper.py

```diff
@@ -72,7 +72,6 @@
     "scalar_name_to_pytorch",
```

Comment:
Is there a reason we still keep it?

Reply:
I was hoping to create less disruption since the names are public. Offline notes: working to find a good way to deprecate them.

Reply:
Removed all